### ISSUE TITLE
popup description of a feature geometry

### ISSUE THREAD

**Issue by edzer** [environmentalinformatics-marburg/mapview#71]
Right now, the feature popup window gives a description of the geometry like `object of class sfc_POINT`; technically, this is not correct as the list-column has this class and the feature geometry is (in this case) an object of class `"XY", "POINT", "sfg"` -- however you don't want to bother users with that. 

I suggest to give it the abbreviated WKT representation, `POINT(181072 333611)` in this case. You get this by `format(st_geometry(obj)[[i]])`, with `obj` the `sf` object, and `i` the index (row number) of the feature. For the `nc` dataset, e.g. we have
```r
> format(nc$geom[[1]])
[1] "MULTIPOLYGON(((-81.47275543..."
```

**Comment by tim-salabim** [comment]
This was just a quick and dirty workaround to prevent failing on list columns (in general, not only `sfc`). I wasn't aware of the power of `format` which seems to be able to handle a multitude of R list objects. Thanks for this tip. 
Now corrected through 0414446db7a54472ea840d67497a838ef1b58565

**Comment by edzer** [comment]
Great, thanks!
